README.md

```diff
@@ -62,21 +63,21 @@ $ az account set --subscription="SUBSCRIPTION_ID"
 
 To execute the Terraform script:
 
-1. **Prepare .tf file**
+### 1. Prepare .tf file
```

Comment:
heading not required since they are three steps that need to be followed.

Reply:
checked in the markdown, these are the most important and the only steps they need to do but unfortunately they font looks small and they might miss it. We had complains before from a couple of our customers that our integration steps were not clear. Just trying to highlight the most important steps required for integration.